server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -126,14 +125,21 @@ public static Club getClub(DatabaseClient dbClient, String clubId) {
           .readRow(
             "Books",
```

Comment:
This also slipped through my last PR so again, totally on me, but this should be "Clubs" not "Books" (otherwise it's grabbing from the wrong table in the DB)

Reply:
Good catch! No need to change it yourself, I've pushed the required change.